_posts/2018-06-22-pulp_qe_intern_update.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 2018 Pulp QE Intern Update #2
+author: Ragavendran Balakrishnan
+tags:
+  - pulpqe
+---
+
+## Journey so Far
+
+The first two weeks of my Red Hat journey was a roller coaster ride with lots of take in's, which I
+have covered in my [first blog](https://pulpproject.org/2018/06/08/pulp_qe_intern_update/). Now,
+Here I am in the third week of my Internship and I have started writing my second blog post  on my
```

Comment:
nitpick - two spaces after post. Also "Here" and "Internship" should be lowercase.